spotinst/ocean_aks_np_node_pool_properties/fields_spotinst_ocean_aks_np_node_pool_properties.go

```diff
@@ -396,3 +493,43 @@ func expandSubnetList(data interface{}) ([]string, error) {
 	}
 	return result, nil
 }
+
+func expandLinuxOSConfig(data interface{}) (*azure_np.LinuxOSConfig, error) {
+	if list := data.([]interface{}); len(list) > 0 {
+		linuxConfig := &azure_np.LinuxOSConfig{}
+		if list[0] != nil {
+			m := list[0].(map[string]interface{})
+			if v, ok := m[string(Sysctls)]; ok {
+				sysctls, err := expandSysctls(v)
+				if err != nil {
+					return nil, err
+				}
+				if sysctls != nil {
+					linuxConfig.SetSysctls(sysctls)
+				} else {
+					linuxConfig.SetSysctls(nil)
+				}
+			}
+		}
+		return linuxConfig, nil
+	}
+	return nil, nil
+}
+
+func expandSysctls(data interface{}) (*azure_np.Sysctls, error) {
+	if list := data.([]interface{}); len(list) > 0 {
+		sysctls := &azure_np.Sysctls{}
+		if list[0] != nil {
+			m := list[0].(map[string]interface{})
+			if v, ok := m[string(VmMaxMapCount)].(int); ok {
+				if v == -1 {
```

Comment:
default is always set to 0 if customer is not provided but here we're checking for -1